utils.lua

```diff
@@ -50,74 +83,89 @@ function util:GetRealmSlug(realm)
     return realmSlug or realm
 end
 
-local unitIdentifiers = {"mouseover", "player", "target", "focus", "pet", "vehicle"}
+local unitIdentifiers = {
+    ["mouseover"] = true,
+    ["player"] = true,
```

Comment:
This was used as a map, and is now working as one: `unitIdentifiers[identifier]`